.github/workflows/nextjs.yml

```diff
@@ -35,9 +35,6 @@ jobs:
         id: detect-package-manager
```

Comment:
Can we remove this?

![image](https://github.com/OpenBCca/openbc-web/assets/40151580/60919e8a-7b93-45ce-a591-b42e93e46533)

Reply:
```suggestion
```

I find [GitHub Suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) great for small changes like this, not sure how widely known it is!

Reply:
I have removed the id token: write.